draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,28 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers {#subscription-id}
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a track, it associates a Request ID with the request.  The
+corresponding Subscription Request ID is equal to `Request ID << 1`.
+
+MoQT also allows the publisher to choose an identifier for each track, called
+the Publisher-Chosen Subscription ID (PSID). PSIDs are odd and have a maximum
+value of 2^61-1. The same identifier MUST NOT be assigned to multiple tracks in the
+same session.  PSIDs are required when the publisher initiates the subscription
+(TODO: delete this; PUBLISH message is forthcoming) and are optional when the
+subscriber initiates it.
+
+When a publisher sends Subgroups or Datagrams, these can contain either a
+Subscription Request ID or Publisher-Chosen Subscription ID, determined by the
+least significant bit of the identifier. Control messages for an established
+subscription can also use either identifier.  Messages that control the
+subscription such as SUBSCRIBE_UPDATE, UNSUBSCRIBE and SUBSCRIBE_DONE also use
+Subscription IDs.
```

Comment:
This seems to be a downgrade from the current draft, which only allows Subscribe ID in those.